angular.json

```diff
@@ -143,9 +143,13 @@
           },
           "configurations": {
             "production": {
-              "browserTarget": "ew-boilerplate-ui:build:production"
```

Comment:
Well, running `ng serve --prod` causes here an error. You shouldn't change project name here. If you want change a project name, then you need to change it everywhere in this file.

Reply:
yes it was my mistake fixed